runtime/compiler/env/j9method.cpp

```diff
@@ -6175,7 +6200,6 @@ TR_ResolvedJ9Method::stringConstant(I_32 cpIndex)
 void *
 TR_ResolvedJ9Method::methodTypeConstant(I_32 cpIndex)
    {
```

Comment:
The ASSERT has been removed from `TR_ResolvedJ9Method::methodTypeConstant(I_32 cpIndex)`